source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md

```diff
@@ -30,6 +30,10 @@ Use the required values from the **Connection Info** tab, the desired environmen
 ```
 ssh -f -N -L PORT:localhost:PORT -p 2222 ENV.SITE_UUID@dbserver.ENV.SITE_UUID.drush.in
 ```
+PORT is the database port specified in the **Connection Info** tab.
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>

```

Comment:
Suggest moving this note to the troubleshooting section below or to include it within the above statement - to reduce clutter of notes. @nataliejeremy what do you think?